015-phpstorm.md

```diff
@@ -34,3 +36,12 @@ registerArgumentsSet(
   \Yiisoft\Http\Method::OPTIONS,
 );
 ```
+
+## PhpStorm attributes
+
+In code CAN use [PhpStorm attributes](https://github.com/JetBrains/phpstorm-attributes), but package 
+`jetbrains/phpstorm-attributes` MUST be added as a dev dependency:
```

Comment:
Wouldn't that result errors on production, when some tools want to process these attributes using `Reflection`?